recipes/gnu-getopt/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+export CFLAGS="-I${PREFIX}/include"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib -L${PREFIX}/include -lintl"
+
+cd "$SRC_DIR"
+make
+make install prefix=$PREFIX
+
+# rename to prevent collision with system (BSD) getopt on OSX
+mv $PREFIX/bin/getopt $PREFIX/bin/gnu-getopt
```

Comment:
Seems like if this is a concern one can simply not install this package and use the system copy, right? Or am I missing something. 🤔

Reply:
The BSD version of getopt is significantly divergent from the GNU version (or rather, vise versa), with different arguments and combatibilities. Some things are written with the BSD version in mind, others with the GNU version. This at leaves it open to alias or symlink to the GNU version, if desired.